src/css/components/_adaptive.scss

```diff
@@ -7,9 +7,9 @@
 
   .vjs-current-time, .vjs-time-divider, .vjs-duration, .vjs-remaining-time,
   .vjs-playback-rate, .vjs-progress-control,
-  .vjs-mute-control, .vjs-volume-control,
```

Comment:
we probably still want to keep `vjs-volume-control` in here and add `vjs-volume-panel` since it's possible to add the volume control to the control bar separately as well.